auth-api/src/auth_api/utils/constants.py

```diff
@@ -20,6 +20,8 @@
 GROUP_GOV_ACCOUNT_USERS = "gov_account_users"
 GROUP_API_GW_USERS = "api_gateway_users"
 GROUP_API_GW_SANDBOX_USERS = "api_gateway_sandbox_users"
+GROUP_MAXIMUS_STAFF = "maximus_staff"
+GROUP_CONTACT_CENTRE = "contact_centre"
```

Comment:
contact_centre_staff.

And as discussed I am okay with these being constants instead of creating the new column, not too much value since we need to check against org_type anyways for now